tests/dummy/app/transitions.js

```diff
@@ -1,24 +1,77 @@
 // import { animate, Promise } from "liquid-fire";
-
 export default function () {
+  let fromRouteIndex = 0
```

Comment:
are these for the addon itself or just demo? Because if these transitions are for the addon then there is a way to package it up so the user doesnt have to copy paste this into their transitions file.